### PR TITLE
passage: init at 0.1.0

### PR DESCRIPTION
**Pull request by rassmike**
###### Motivation for this change
Add passage a password store utilizing the age encryption library.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -10942,4 +10942,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  rassmike = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:10945]: Please put this into a separate commit with the message ``maintainers: add rassmike``.

File: pkgs/tools/security/passage/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{ lib, rustPlatform, installShellFiles, asciidoctor, xorg, python3, passAlias ? false }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "passage";
+  version = "0.1.0";
+
+  src = builtins.fetchGit {
```
Comment (SuperSandro2000) [inline pkgs/tools/security/passage/default.nix:7]: Please use fetchFromSourcehut instead.

File: pkgs/tools/security/passage/default.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    url = "https://git.sr.ht/~gpanders/passage";
+    ref = "master";
+    rev = "9bddfe8e03b39ceccfa736e88924d3b4cb59748f";
+  };
+
+  cargoSha256 = "06clfavpxc7a11jxml0vkw8x7lya5kipv8xyi59f8fqsksj79k8m";
+
+  nativeBuildInputs = [ asciidoctor python3 installShellFiles ];
+  buildInputs = [ xorg.libxcb.dev ];
```
Comment (SuperSandro2000) [inline pkgs/tools/security/passage/default.nix:16]: Did you try if xorg.libxcb is enough? dev should be added automatically.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (rassmike) [reply maintainers/maintainer-list.nix:10945]: DONE, sorry about that.

File: pkgs/tools/security/passage/default.nix (hunk 2 above)
Comment (rassmike) [reply pkgs/tools/security/passage/default.nix]: Fixed it, now it uses fetchFromSourcehut.

File: pkgs/tools/security/passage/default.nix (hunk 3 above)
Comment (rassmike) [reply pkgs/tools/security/passage/default.nix]: Yeah, you're right xorg.libxcb is indeed enough. Hadn't checked.
